lib/components/user/saved-trip-screen.js

```diff
@@ -93,49 +94,12 @@ class SavedTripScreen extends Component {
 
   /**
    * Saves a trip and returns to the saved trips screen.
+   * On success, this operation will also make the browser
+   * navigate to the Saved trips page.
    * @param {*} monitoredTrip The trip edited state to be saved, provided by Formik.
    */
-  _handleDeleteTrip = async () => {
-    if (confirm('Would you like to remove this trip?')) {
-      const { deleteUserMonitoredTrip } = this.props
-      const { monitoredTrip } = this.state
-      await deleteUserMonitoredTrip(monitoredTrip.id)
-
-      this._goToSavedTrips()
-    }
-  }
-
-  _handleSaveNewTrip = async () => {
-    await this._updateMonitoredTrip()
-
-    // For new trips, if a trip with the same ui_activeSearch value in queryParams
-    // appears in the monitored trips redux state,
-    // then we know that save was successful and we can proceed.
-    const { activeSearchId } = this.props
-
-    const savedTrip = this.props.monitoredTrips.find(trip => {
-      const tripQueryParams = qs.parse(trip.queryParams.split('?')[1])
-      return tripQueryParams.ui_activeSearch === activeSearchId
-    })
-
-    if (savedTrip) {
-      this._goToTripPlanner()
-    }
-  }
-
-  _handleSaveTripEdits = async () => {
-    const { monitoredTrip } = this.state
-
-    await this._updateMonitoredTrip()
-
-    // For existing trips, if the lastUpdated value (long) for the trip with the same id in monitored trips redux state
-    // is more recent that our copy, then we know that save was successful and we can proceed.
-    // (The state is not updated upon save because the next normal step is to close this screen.)
-    const savedTrip = this.props.monitoredTrips.find(trip => trip.id === monitoredTrip.id)
-
-    if (savedTrip.lastUpdated > monitoredTrip.lastUpdated) {
-      this._goToSavedTrips()
-    }
+  _handleSaveTrip = async monitoredTrip => {
+    await this._updateMonitoredTrip(monitoredTrip)
   }
```

Comment:
`_handleSaveTrip` does not need to exist (just use `_updateMonitoredTrip`). Also, `_updateMonitoredTrip` does not need to be async (no need to await `createOrUpdateUserMonitoredTrip`).

Reply:
Good catch. Updated in https://github.com/opentripplanner/otp-react-redux/pull/232/commits/3fd0a0ab2a63ee98522c108e5e31ae031777b83c along with missing import in `actions/user`.